composer.json

```diff
@@ -29,6 +29,7 @@
         "symfony/options-resolver": "^2.7 || ^3.0"
     },
     "require-dev": {
+        "defuse/php-encryption":   "^2.0",
```

Comment:
I have no idea. =)

I trust that you have done some research to find a popular and robust package. And also one that you like.